GameServerConsole/GameServerConsole.csproj

```diff
@@ -10,7 +10,7 @@
     <Prefer32Bit>false</Prefer32Bit>
     <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
     <Optimize>false</Optimize>
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x86</PlatformTarget>
```

Comment:
This isn't regarding the issue but it resolves build settings until we have x64 libenet